### ISSUE TITLE
If a post has words inside <>, they are hidden or not shown in astral.ninja 

### ISSUE THREAD

**Issue by vishalxl** [monlovesmango/astral#99]
Since the text is supposed to be plain text ( except for mentions etc), such deletion or processing should not ideally be done. 

Same kind of behavior is seen in nostr.io just for the record. and I guess many other web clients.

**Comment by monlovesmango** [comment]
yes bc I think allowing this opens you up to xss attacks in javascript runtime env. so its really a feature not a bug. 

do you have any documentation on why it is safe to display this? everything I found seemed to say its not safe.

**Comment by vishalxl** [comment]
I'm only looking at from UI perspective, that once you have verified that this is not an issue/attack, you may display it. 

one could even replace every < with [ and > with ] and then display it, if that helps.

**Comment by vishalxl** [comment]
Making sure that the string is not an attack is of primary importance. I am just highlighting that after that has been done, and maybe the characters are removed/modified/escaped if needed, then the content should get displayed. 

I take it there would be way to escape the character if nothing else works, or just remove it if needed, to make it less dangerous.  

Worse case scenario: a removed <> characters would be better than nothing imo. 

Its not  a high priority thing, and doubt gets used a lot, but I recently got into it thats why I raised this.
